### PR TITLE
Bug Fixes, error handling, typography consistency

### LINKED ISSUE #18: make every instance of Roll number to `rollNo` for consistency

### PR DESCRIPTION
**Pull request by 1-Harshit**
* Fixed bugs in redeem
* Fixes Error messages
* Fixed pq search query by refactoring const string as variable
* Resolves #18

### CODE REVIEW

File: database/connect.go [hunk 1]
```diff
@@ -21,7 +21,9 @@ func Connect() (err error) {
 	)
 
 	psqlconn := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s", host, port, user, password, dbname)
-    
+    log.Info("Connecting to database", psqlconn)
+
+	psqlconn = "postgres://mkeopogsvhqncb:c1fbd9d5bc2c043e353198280ed0fb1de1402c5b3dbec6ce98927628d3b6bbf1@ec2-52-200-68-5.compute-1.amazonaws.com:5432/d3orbffmu420pg"
```
Comment (bhuvansingla) [inline database/connect.go]: No hard coding here,

File: handlers/history.go [hunk 2]
```diff
@@ -20,29 +20,27 @@ func WalletHistory(w http.ResponseWriter, r *http.Request) error {
 		return errors.NewHTTPError(nil, http.StatusMethodNotAllowed, http.StatusText(http.StatusMethodNotAllowed))
 	}
 
-	queriedRollno := r.URL.Query().Get("rollno")
+	queriedRollNo := r.URL.Query().Get("rollno")
```
Comment (bhuvansingla) [inline handlers/history.go]: rollNo

File: server/router.go [hunk 3]
```diff
@@ -15,14 +15,16 @@ func setRoutes() {
 	http.HandleFunc("/auth/check", CORS(auth.IsAuthorized((errors.Handler(handlers.CheckLogin)))))
 	http.HandleFunc("/auth/otp", CORS(errors.Handler(handlers.GenerateOtp)))
 	http.HandleFunc("/auth/logout", CORS(errors.Handler(handlers.Logout)))
+
```
Comment (bhuvansingla) [inline server/router.go:19]: Why are there two newlines?

**Review by bhuvansingla — CHANGES_REQUESTED** [review]

File: database/connect.go (hunk 1 above)
Comment (1-Harshit) [reply database/connect.go]: Damn

**Review by bhuvansingla — APPROVED** [review]

**Pull request merged**